Code/IO/include/sitkShow.h

```diff
@@ -78,19 +78,15 @@ namespace simple
    *  The string after \c "-eval" is an ImageJ macro the opens the file and runs ImageJ's Make Composite
    *  command to display the image in color.
    *
+   *  Note that the environment variables are only checked when SimpleITK is first launched.
+   *  This behavior is different than previous versions of SimpleITK, where the variables
+   *  were checked every time Show was called.
+   *
    *  If the \c "%f" token is not found in the command string, the temporary file name is automatically
```

Comment:
I think the comment above "This behavior is different..." should be removed. New version of SimpleITK, new behavior. Documentation is for the current version, these notes should be added to the release notes with all the other changes.

Reply:
IMHO, much of this documentation should not be duplicated form the ImageView. The documentation here should say that it is a procedural wrapper for the ImageViewer class, and details of behavior should be documented *only* there.

Reply:
Yeah, that would make sense